PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/projects/HDInsightModuleBuilder.java

```diff
@@ -47,19 +50,30 @@
 import org.jetbrains.annotations.NotNull;
 
 import javax.swing.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
 
 public class HDInsightModuleBuilder extends JavaModuleBuilder implements ModuleBuilderListener {
     private HDInsightTemplateItem selectedTemplate;
     private LibrariesContainer librariesContainer;
     private LibraryCompositionSettings scalaLibraryCompositionSettings;
     private SparkVersion sparkVersion;
+    private ScalaPluginStatus scalaPluginStatus;
+
+    private static final String SCALA_PLUGIN_ID = "org.intellij.scala";
+    private static final String SCALA_PLUGIN_INSTALL_MSG = "<HTML>No Scala Intellij plugin found. " +
```

Comment:
Thanks for the suggestion. The `html` in `String` is more readable. The `SCALA_PLUGIN_INSTALL_MSG ` is not dynamic. We use the html to emphasize the `click` word in the string. Use `String` is enough. Let's keep it.